integration-tests/config-key/src/main/resources/application.properties

```diff
@@ -0,0 +1,15 @@
+# custom_config_key
+quarkus.rest-client.custom_config_key.url=http://localhost:8080
+quarkus.openapi-generator.codegen.spec.config_key_openapi_yaml.config-key=custom_config_key
+quarkus.openapi-generator.codegen.spec.custom_config_key.mutiny=true
+
+# use the @AnotherCustomAnnotation with `config_key_openapi_yaml`
+quarkus.openapi-generator.codegen.spec.config_key_openapi_yaml.additional-api-type-annotations=@io.quarkiverse.openapi.generator.configkey.AnotherCustomAnnotation
+quarkus.openapi-generator.codegen.spec.custom_config_key.additional-api-type-annotations=@io.quarkiverse.openapi.generator.configkey.CustomAnnotation
+
+
+# empty config-key
+quarkus.openapi-generator.codegen.spec.empty_config_key_yaml.config-key=
+
+quarkus.openapi-generator.codegen.spec.empty_config_key_yaml.additional-api-type-annotations=@io.quarkiverse.openapi.generator.configkey.CustomAnnotation
+quarkus.rest-client.empty_config_key_yaml.url=http://localhost:8080
```

Comment:
I don't see any tests for these configs. Did you forget them?

Reply:
Hi @hbelmiro TY for the review, I added one integration test to certify if when the `config-key` is empty will get the default.